package.json

```diff
@@ -2020,6 +2020,16 @@
                     }
                 }
             },
+            {
```

Comment:
I had a previous conversation with Justin and it might be a good idea to contribute directly to `master` to avoid the `feature/*` branch merging process completely since this feature does not need to be hidden.

For the command palette command I used `"when": "aws.isDevMode"` for my command `aws.auth.showConnectionsPage` in package.json

Reply:
The "when" tells the toolkit what conditions must be met to show the command right? But what does the `"aws.isDevMode"` condition mean?

Reply:
Add `"aws.dev.forceDevMode": true` to your VS Code settings.